### PR TITLE
Add batch sending items

### PR DESCRIPTION
**Pull request by IamDejv**
## Description

Describe your changes in detail.

## Motivation and context

Why is this change required? What problem does it solve?

If it fixes an open issue, please link to the issue here (if you write `fixes #num`
or `closes #num`, the issue will be automatically closed when the pull is accepted.)

## How has this been tested?

Please describe in detail how you tested your changes.

Include details of your testing environment, and the tests you ran to
see how your change affects other areas of the code, etc.

## Types of changes

What types of changes does your code introduce? Put an `x` in all the boxes that apply:
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

Go over all the following points, and put an `x` in all the boxes that apply.

Please, please, please, don't send your pull request until all of the boxes are ticked. Once your pull request is created, it will trigger a build on our [continuous integration](http://www.phptherightway.com/#continuous-integration) server to make sure your [tests and code style pass](https://help.github.com/articles/about-required-status-checks/).

- [ ] I have read the **[CONTRIBUTING](CONTRIBUTING.md)** document.
- [ ] My pull request addresses exactly one patch/feature.
- [ ] I have created a branch for this patch/feature.
- [ ] Each individual commit in the pull request is meaningful.
- [ ] I have added tests to cover my changes.
- [ ] If my change requires a change to the documentation, I have updated it accordingly.

If you're unsure about any of these, don't hesitate to ask. We're here to help!

### CODE REVIEW

File: src/Resource/BatchSendingItem.php [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+use DateTime;
+use DigitalCz\DigiSign\Resource\Traits\EntityResourceTrait;
+
+class BatchSendingItem extends BaseResource
+{
+    use EntityResourceTrait;
+
+    public string $name;
+
+    public int $itemNumber;
```
Comment (PavelVais) [inline src/Resource/BatchSendingItem.php]: tohle bych přejmenoval už v API response z Digisign prostředí a název upravil

File: src/Resource/BatchSendingItem.php [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+    public ?string $envelope;
```
Comment (PavelVais) [inline src/Resource/BatchSendingItem.php]: přejmenovat na `$envelopeId`

**Review by PavelVais — CHANGES_REQUESTED** [review]

File: src/Resource/BatchSendingItem.php (hunk 2 above)
Comment (IamDejv) [reply src/Resource/BatchSendingItem.php]: envelope je ve skutecnosti Envelope, zapomnel jsem zmenit

File: src/Resource/BatchSendingItem.php (hunk 1 above)
Comment (IamDejv) [reply src/Resource/BatchSendingItem.php]: Odstraneno z API, dle komentare to je pro Debug, tedy nebudeme zverejnovat do SDK

File: CHANGELOG.md [hunk 3]
```diff
@@ -26,6 +26,7 @@ Updates should follow the [Keep a CHANGELOG](http://keepachangelog.com/) princip
 - Add `Envelope.createdBy` and `Envelope.sentBy`
 - Add `BatchSendings` and `BatchSendingsItems` endpoints
 - Add `AccountSecurity.localSignature` to resource
+- Add `BatchSendingItems` endpoints and resources
```
Comment (PavelVais) [inline CHANGELOG.md]: tohle je možné odstranit, již je to zmíněno na řádku 27 🤔

**Review by PavelVais — APPROVED** [review]

**Pull request merged**